### DISCUSSION TITLE
Cant view notebooks in repository

### DISCUSSION THREAD

**Discussion by anishjohnson** [community/community#23620] [Other Feature Feedback, Questions, & Ideas]
<p>I am working on a project and when I save a copy in the (Private repository) repository using colab and try to view it later it keeps on showing An error occurred,  I tried creating new repo as well as creating new notebook using colab and still can view my notebook. This problem started recently, but I am able to view my previous colab notebooks in other repositories. Any help would be appriciated.</p>

**Comment by jfullstackdev** [comment]
<p>did you follow the steps? you need to manually check that option for private repos,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/e/fe169fbb1b277bfdde231b2c358fb7cd452fe683.png" data-download-href="/uploads/short-url/AfLKrsUBSt6zKayndSAquNULakr.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181078258-6df14b33-3e7a-45e4-bd54-3d0a0434a9e6.png" alt="image" data-base62-sha1="AfLKrsUBSt6zKayndSAquNULakr" width="690" height="187" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/e/fe169fbb1b277bfdde231b2c358fb7cd452fe683_2_690x187.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/e/fe169fbb1b277bfdde231b2c358fb7cd452fe683_2_1035x280.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/e/fe169fbb1b277bfdde231b2c358fb7cd452fe683_2_1380x374.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/e/fe169fbb1b277bfdde231b2c358fb7cd452fe683_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1714×465 48 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by anishjohnson** [comment]
<p>Yes, I have done this previously and have accessed my notebooks via repos but this time I am facing issues.</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="anishjohnson" data-post="3" data-topic="219797">
<div class="title">
<div class="quote-controls"></div>
 anishjohnson:</div>
<blockquote>
<p>I have done this previously and have accessed my notebooks via repos</p>
</blockquote>
</aside>
<p>including private repos?</p>
<p>and also there is an on-going problem at GitHub and even though others are not related to GitHub Actions, others report problems not related to it,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/4/84b09d7380312d93739af263e4d288b9877ff26a.png" data-download-href="/uploads/short-url/iVPqauI4lkosYFltmtDWFbjUA6m.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181078261-310dbd57-a5b6-4c95-9b7d-f2dda918b5fa.png" alt="image" data-base62-sha1="iVPqauI4lkosYFltmtDWFbjUA6m" width="690" height="119" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84b09d7380312d93739af263e4d288b9877ff26a_2_690x119.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84b09d7380312d93739af263e4d288b9877ff26a_2_1035x178.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84b09d7380312d93739af263e4d288b9877ff26a_2_1380x238.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/4/84b09d7380312d93739af263e4d288b9877ff26a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1525×264 34.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p><a href="https://www.githubstatus.com/" rel="noopener nofollow ugc">https://www.githubstatus.com/</a></p>

**Comment by anishjohnson** [comment]
<p>Yes, I have saved colab notebooks for previous projects also the private ones, I too came across this problem GitHub is going through but I am facing this same issue for the past week.</p>

**Comment by jfullstackdev** [comment]
<p>how about public repos?</p>

**Comment by anishjohnson** [comment]
<p>All the other repos are working fine even the private ones, just the ones I created last week are facing these issues.</p>

**Comment by jfullstackdev** [comment]
<p>I’m checking it also, it’s working fine, but I cannot see your private repo, only this public repo</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/3/d3eb7ff80b0a30a9cefa666cd4e460c43ba347dd.png" data-download-href="/uploads/short-url/ueJiKwYmTtS7BLhcmky5yP2NXbf.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181078263-629c79ae-60d1-4c98-98f4-064725e4c680.png" alt="image" data-base62-sha1="ueJiKwYmTtS7BLhcmky5yP2NXbf" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3eb7ff80b0a30a9cefa666cd4e460c43ba347dd_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3eb7ff80b0a30a9cefa666cd4e460c43ba347dd_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3eb7ff80b0a30a9cefa666cd4e460c43ba347dd_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/3/d3eb7ff80b0a30a9cefa666cd4e460c43ba347dd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 147 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>I’m wondering why it will not work at your private repo when it’s working at the public one,</p>
<p>is it possible this might be on the side of the app itself? not here on GitHub?</p>

**Comment by anishjohnson** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/6/96ae2fa77dfe01381b6625dd374b02a03456e943.jpeg" data-download-href="/uploads/short-url/luYOnUmHfnETwnUipu24VERsdqj.jpeg?dl=1" title="repo4" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/87002524/181078265-28e106c5-4186-4905-a5fc-2025ff3a5bc5.jpeg" alt="repo4" data-base62-sha1="luYOnUmHfnETwnUipu24VERsdqj" width="504" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/6/96ae2fa77dfe01381b6625dd374b02a03456e943_2_504x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/9/6/96ae2fa77dfe01381b6625dd374b02a03456e943_2_756x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/9/6/96ae2fa77dfe01381b6625dd374b02a03456e943_2_1008x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/6/96ae2fa77dfe01381b6625dd374b02a03456e943_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">repo4</span><span class="informations">1257×1245 251 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>The first two repos are working fine just facing issues with the third one.</p>

**Comment by jfullstackdev** [comment]
<p>yup, I’m now seeing more, can you check that at the Developer Tools?</p>
<p>for example your public repo,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/2/e2193c95d0ec91c22c7d9ac1b647632d6ff3490b.png" data-download-href="/uploads/short-url/wg9YBXP4yyI0xRpkxbmqdXFsNQv.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181078267-6e729e39-13d4-427c-ae35-121790b55d8b.png" alt="image" data-base62-sha1="wg9YBXP4yyI0xRpkxbmqdXFsNQv" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2193c95d0ec91c22c7d9ac1b647632d6ff3490b_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2193c95d0ec91c22c7d9ac1b647632d6ff3490b_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2193c95d0ec91c22c7d9ac1b647632d6ff3490b_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/2/e2193c95d0ec91c22c7d9ac1b647632d6ff3490b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 180 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>if you don’t know, click the 3 dots in Chrome, upper right corner, go to <code>More tools</code> and click <code>Developer Tools</code> then click <code>Console</code>,</p>
<p>as you can see here, it does not have any error because it’s working fine, we will be seeing what is not loading at your private repo, if ever there is,</p>

**Comment by anishjohnson** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/f/1fb18aad3754f471da1444109618c12dea52dedf.png" data-download-href="/uploads/short-url/4wnblcfjrcEW13qox8A2BNckVQz.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/87002524/181078268-32781284-617f-46c5-b7ea-cf04eb7f9290.png" alt="image" data-base62-sha1="4wnblcfjrcEW13qox8A2BNckVQz" width="638" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1fb18aad3754f471da1444109618c12dea52dedf_2_638x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1fb18aad3754f471da1444109618c12dea52dedf_2_957x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1fb18aad3754f471da1444109618c12dea52dedf_2_1276x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1fb18aad3754f471da1444109618c12dea52dedf_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2091×1638 77.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
Getting this error

**Comment by jfullstackdev** [comment]
<p>can you click the link? the <code>notebooks. ...</code></p>
<p>from that point we know that’s the error, it cannot load your specific notebook, it’s 500 Internal Server Error</p>

**Comment by anishjohnson** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/5/85bad1ec58b006c26cd4ab9a69527ca01bff83d8.png" data-download-href="/uploads/short-url/j51LlhrqXBn30OHbRM7zEf7J9Bm.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/87002524/181078270-1a972b61-411d-4fc8-951b-b3a31df58559.png" alt="image" data-base62-sha1="j51LlhrqXBn30OHbRM7zEf7J9Bm" width="603" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85bad1ec58b006c26cd4ab9a69527ca01bff83d8_2_603x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85bad1ec58b006c26cd4ab9a69527ca01bff83d8_2_904x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85bad1ec58b006c26cd4ab9a69527ca01bff83d8_2_1206x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/5/85bad1ec58b006c26cd4ab9a69527ca01bff83d8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2082×1725 203 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>can you click the <code>83b4 ... </code></p>
<p>it will be expanded when you click that</p>

**Comment by anishjohnson** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/c/7c9d6caa4d20171434e802eba934c495f96bcb75.png" data-download-href="/uploads/short-url/hMouaVLSHzA3gap9ZXjY4FELsWh.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/87002524/181078272-820f753f-78d3-4c77-8bb4-0f9309e4527c.png" alt="image" data-base62-sha1="hMouaVLSHzA3gap9ZXjY4FELsWh" width="662" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7c9d6caa4d20171434e802eba934c495f96bcb75_2_662x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7c9d6caa4d20171434e802eba934c495f96bcb75_2_993x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7c9d6caa4d20171434e802eba934c495f96bcb75_2_1324x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/c/7c9d6caa4d20171434e802eba934c495f96bcb75_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2280×1722 239 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>now it is expanded, click again the link at the bottom, it will go to that specific file hopefully,</p>
<p>but again, we already know the error is 500, that specific file is not loading, maybe this time, hopefully it will be fixed,</p>
<p>or try to create a new one to test whether it is successful</p>

**Comment by anishjohnson** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/8/b840901423859eed3956d58efb68d56f5ec79399.png" data-download-href="/uploads/short-url/qhYb8UUjYeO81T0GX3KveQThRyx.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/87002524/181078273-f12d4e2b-739b-4daf-a30b-ee0939f63f29.png" alt="image" data-base62-sha1="qhYb8UUjYeO81T0GX3KveQThRyx" width="670" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b840901423859eed3956d58efb68d56f5ec79399_2_670x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b840901423859eed3956d58efb68d56f5ec79399_2_1005x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b840901423859eed3956d58efb68d56f5ec79399_2_1340x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b840901423859eed3956d58efb68d56f5ec79399_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2280×1701 244 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment, marked as answer]
<p>try to test it creating another one public and one private with plain text,</p>
<p>if that is working both, then maybe, it was simply that specific file that is the error,</p>
<p>just like there are characters that are not allowed or links at the file that also refused to connect etc.,</p>

**Comment by anishjohnson** [comment]
<p>Thanks for all the help, it seems the problem is with the notebook. I tried the same with a new notebook in private as well as public repos and they work fine. So I will have to check with my code.<br>
Thanks for your time.</p>
